src/Robo/Commands/Blt/UpdateCommand.php

```diff
@@ -314,8 +314,8 @@ protected function convertLegacySchemaVersion($version) {
    * @throws BltException
    */
   protected function rsyncTemplate() {
-    $source = $this->getConfigValue('blt.root') . '/template';
-    $destination = $this->getConfigValue('repo.root');
+    $source = 'vendor/acquia/blt/template';
```

Comment:
@danepowell what is the impact of this? Isn't BLT always run from the root project directory? Do you need full paths?

On Windows, the problem is rsync does not work with full paths as it's running under cygwin, so whatever php returns as root path, it's different on cygwin: c:/ is /cygdrive/c/ for example.

Reply:
_Usually_ folks run it from the project root, but we support running it from anywhere below the project root. I don't think we want to break that support.

This is the function that discovers the BLT root, you can try to make it relative. Although it makes me pretty nervous to change this fundamental.

https://github.com/acquia/blt/blob/10.x/src/Robo/Config/DefaultConfig.php#L28-L45

Reply:
Got it, I'll revisit and try to just patch for Windows.